include/routes.php

```diff
@@ -157,10 +157,10 @@
         $feather->get('/ip-stats/id/:id(/)', '\controller\admin\users:ipstats')->conditions(array('id' => '[0-9]+'));
         $feather->get('/show-users/ip/:ip(/)', '\controller\admin\users:showusers');
     });
-    
+
 });
 
 // 404 not found
-$feather->notFound(function () use ($lang_common) {
-    message($lang_common['Bad request'], '404');
+$feather->notFound(function () use ($feather) {
+    $feather->halt(404, 'Not found');
```

Comment:
All right :) It was meant to be a temporary solution, while waiting for a better error page which uses Slim interface